lerna.json

```diff
@@ -9,7 +9,7 @@
     "publish": {
       "allowBranch": [
         "e2e",
```

Comment:
should we remove e2e?

Reply:
https://github.com/Zilliqa/zilliqa-js/blob/main/tasks/local-registry.sh#L12
We can't remove it. It's for e2e testing.